bindings/python/src/lib.rs

```diff
@@ -26,10 +31,10 @@ pub(crate) fn block_on<C: futures::Future>(cb: C) -> C::Output {
     runtime.lock().unwrap().block_on(cb)
 }
 
-/// Init the logger of wallet library.
+/// Init the Rust logger.
 #[pyfunction]
-pub fn init_logger(config: String) -> PyResult<()> {
-    rust_init_logger(config).expect("failed to init logger");
+pub fn init_logger(config: String) -> Result<()> {
+    rust_init_logger(config).map_err(|err| Error::from(format!("{:?}", err).as_str()))?;
```

Comment:
We probably should add a `From<String>` impl for `Error`. We have a string here (format!) only to pass it as a `&str` to then turn it into a `String` again.